.code-samples.meilisearch.yaml

```diff
@@ -77,16 +77,62 @@ search_post_1: |-
 search_get_1: |-
   curl \
     -X GET 'http://localhost:7700/indexes/movies/search?q=american%20ninja'
-get_update_1: |-
+get_task_by_index_1: |-
   curl \
-    -X GET 'http://localhost:7700/indexes/movies/updates/1'
-get_all_updates_1: |-
+    -X GET 'http://localhost:7700/indexes/movies/tasks/1'
+get_all_tasks_1: |-
   curl \
-    -X GET 'http://localhost:7700/indexes/movies/updates'
-get_keys_1: |-
+    -X GET 'http://localhost:7700/tasks'
+get_task_1: |-
   curl \
-    -H "X-Meili-API-Key: 123" \
-    -X GET 'http://localhost:7700/keys'
+    -X GET 'http://localhost:7700/tasks/1'
+get_all_tasks_by_index_1: |-
+  curl \
+    -X GET 'http://localhost:7700/indexes/movies/tasks'
+get_one_key_1: |-
+  curl \
+    -X GET 'http://localhost:7700/keys/d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4' \
+    -H 'Authorization: Bearer yourMasterKey'
+get_all_keys_1: |-
+  curl \
+    -X GET 'http://localhost:7700/keys' \
+    -H 'Authorization: Bearer yourMasterKey'
+create_a_key_1: |-
+  curl \
+    -X POST 'http://localhost:7700/keys' \
+    -H 'Authorization: Bearer yourMasterKey' \
+    -H 'Content-Type: application/json' \
+    --data-binary '{
+      "description": "Add documents: Products API key",
+      "actions": [
+        "documents.add"
+      ],
+      "indexes": [
+      "products"
+      ],
+      "expiresAt": "2021-11-13T00:00:00Z"
+    }'
+update_a_key_1: |-
+  curl \
+    -X PATCH 'http://localhost:7700/keys/d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4' \
+    -H 'Authorization: Bearer yourMasterKey' \
+    -H 'Content-Type: application/json' \
+    --data-binary '{
+      "description": "Manage documents: Products/Reviews API key", // Update key description to specify its new scope
+      "actions": [
+        "documents.add",
+        "documents.delete" // Add access to document deletion
+      ],
+      "indexes": [
+        "products",
+        "reviews" // Add access to the "reviews" index
```

Comment:
Are you aware that copying/pasting this curl example won't work since the comments are not accepted in JSON?
The users will have to delete the comment before sending the request.

Reply:
Thanks for noticing that @curquiza . I think we always want users to be able to copy/paste API reference code samples, so I'm removing the comments.